### ISSUE TITLE
Does a callback e.g. onCurrentItemChanged exist?

### ISSUE THREAD

**Issue by flolege** [rrag/react-stockcharts#750]
I have a Candlestick chart.

 I want to get a callback when the mouse pointer moves from one candle to another one. 

My current workaround is (ab)using the HoverTooltip tooltipContent prop where I do something like this, using a global var currentItemSelected:

`
		tooltipContent = ()=> {
		const dateFormat = timeFormat("%Y-%m-%d");
		const numberFormat = format(".2f");

		return ({ currentItem, xAccessor }) => {
			
			if(currentItemSelected == {} || currentItemSelected.startTime != currentItem.startTime) {
				currentItemSelected = currentItem
				this.props.onCurrentItemUpdated(currentItem)
			}
....			
`
Is there a better way to achieve what I want?

**Comment by flolege** [comment]
As it often is, I found (something close to) a solution after posting the issue: the ClickCallback

I can listen to onMouseMove events here.

Unfortunately, anything in that callback causes high CPU load. I optimized my code to not store a global object but only the timestamp (int) to compare it with the current Item. Even only this comparison with two ints  causes the CPU goes up to 35% as I have many candles.

Is there something even better than the onMosueMove event / the ClickCallback?
